core/main/src/Core/Container.ts

```diff
@@ -393,4 +393,11 @@ export class Container {
 
         this.density = (canvas.width * canvas.height) / (densityOptions.factor * pxRatio * densityOptions.area);
     }
+
+    private loadTheme(name?: string): void {
+        if(name){
+        for(const themeOption in this.options.themes.)
```

Comment:
`for (const themeOption of this.options.themes) {` is the right way, but a shorter way to find that would be nicer like `filter` or `find`